### ISSUE TITLE
overlap issue in main menu

### ISSUE THREAD

**Issue by warengonzaga** [algofilipino/homepage#7]
a visible issue in the main menu see the screenshot below.

![image](https://user-images.githubusercontent.com/15052701/136647280-81fcfdd7-64bd-4326-98cd-1375fa1a7e0f.png)

**Comment by WarenGonzaga** [comment]
Re-opening the issue, the same issue appears after the deployment to Vercel. Can you take a look into it @felixjeromedelafuente?

**Comment by felixjeromedelafuente** [comment]
Hi sir @WarenGonzaga, I inspected the deployed website and also the main and dev branch. It seems that the deployed website uses the main branch while the changes I made in the Navbar were merged on the dev branch because I created a PR for my changes on the dev branch based on the instructions from the ReadMe file.

Navbar.jsx on main branch:
![image](https://user-images.githubusercontent.com/51954937/137259214-4c2cd727-e119-4640-a86d-a26e61933556.png)

Navbar.jsx on dev branch:
![image](https://user-images.githubusercontent.com/51954937/137259299-9a3c170e-c70b-4111-bbfc-bdf3aa732e7c.png)

Should I create a new PR directing it to the main branch instead?
